sycl/include/sycl/ext/oneapi/matrix/static-query-use.hpp

```diff
@@ -206,12 +206,12 @@ struct tpu_params<
 
   template <typename Group>
   using joint_matrix_a =
-      joint_matrix<Ta, defaultM, defaultK, use::a, layout::row_major, Group>;
+      joint_matrix<Ta, use::a, defaultM, defaultK, layout::row_major, Group>;
```

Comment:
tpu_params should now take layout A and layout B as template arguments so we can pass them here.